.travis.yml

```diff
@@ -1,17 +1,12 @@
 language: csharp
 sudo: false
-mono:
-  - 4.6.2 # in most common distros
-  - latest
-os:
-  - linux
-  - osx
 matrix:
-  exclude:
-    - os: osx
+  include:
+    - os: linux
+      mono: 4.6.2
+    - os: linux
       mono: latest
-  allow_failures:
-    - mono: latest
-  fast_finish: true
+    - os: osx
+      mono: 4.6.2
```

Comment:
My assumption is that some people are early adopters and others are not. No particular reason to pick 4.6.2 for OSX, however. I just used it because it's common with Linux. But I'm actually in the  dark in terms of what OSX users may actually want. ☹️